location_register/models/address_models.py

```diff
@@ -13,3 +13,8 @@ class Meta:
         verbose_name = _('country')
         verbose_name_plural = _('countries')
         ordering = ('name',)
+
+    def save(self, *args, **kwargs):
+        self.name = str(self.name).strip()
+        self.name_uk = str(self.name_uk).strip()
+        super(Country, self).save(*args, **kwargs)
```

Comment:
this is python 2 syntax
```suggestion
        super().save(*args, **kwargs)
```